src/method/process.tex

```diff
@@ -2,23 +2,123 @@
 That is the Node.js backend, Stockfish and Firebase Emulator.
 % textidote: ignore end
 To manage them, the team uses Docker~\cite{docker} to simultaneously run all the services.
 Docker is a platform that allows developers to isolate their applications in containers, which are lightweight and can
-run on any machine.
+run on any machine on any operating system.
 This makes it painless to develop multiservice projects, as there are no compatibility issues between the different
 devices and services.
 To handle multiple containers, the team uses Docker Compose, which is a tool for defining and running multi-container
 Docker applications.
+This makes it possible to run all the services locally when developing, simulating how they will behave in a production
+environment.
+
+% textidote: ignore begin
+\subsection{CI/CD}\label{subsec:ci/cd}
+% textidote: ignore end
+
+The team uses GitHub~\cite{github} for version control and continued integration and development (CI/CD) for both the
+report writing process and the development of the software.
+GitHub provides many features, such as pull requests, issues, actions, and projects, which makes it easy to collaborate
+and work on the project and the report.
+
+The team uses branches, where the unwritten rules is that only the creator of the branch makes changes to the code in
+the branch to avoid accidental overwriting of code.
+Having branches makes it simple to track what team members are working on and how far in development they are.
+
+\subsubsection{Task management}
+
+The team makes use of GitHub projects to plan out different tasks and issues to track bugs and features.
+A snapshot of some categories with some issues can be seen in Figure~\ref{fig:github-project}.
+In the figure, it is evident that issues move from category to category as the team progresses with them.
+
+The custom protocol is for a team member to assign one self to an issue and thereafter complete the task associated with
+the issue.
+This way, other team members can comment inside the issue if they have comments about what the issue entails.
+Another reason for this method is that it is clear to all members what problems are being solved in a given pull request
+as the relevant issue is linked, thereby improving efficiency and clarity.
+This method of creating issue and assigning one self to them makes for a lot of freedom in development.
+
+This freedom however comes with the responsibility for team members to adequately assign themselves to enough issues to
+cover their respective workload commitments.
+Lastly, when having the list of issues dynamically update as team members resolves them makes for a clean and
+comprehensible overview of how much work needs to be done, how much is in progress, and how much has been done.
+
+% textidote: ignore begin
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=1\textwidth]{github-projects}
+    \caption{The P2 group 8 GitHub Project overview.}\label{fig:github-project}
```

Comment:
No need to include "P2 group 8" to the caption as the reader won't know what that refrences to exactly. A better replacement would be "The team's..."

Reply:
Agreed. Will fix